### DISCUSSION TITLE
Image Quality Assessment for Magnetic Resonance Imaging - How FID is used?

### DISCUSSION THREAD

**Discussion by windj007** [photosynthesis-team/piq#314] [Q&A] [answered]
Hi @snk4tr, @zakajd!

I've just read your paper on IQA in MRI domain, congrats for the accept, very insightful!

Could you please elaborate more on how distribution-level metrics (e.g. FID) are applied to measure the quality of a single image? Do you split the evaluated image into multiple crops and compare them as-a-set against a fixed set high-quality images? Or any other strategy? Have I just missed the description from the paper (if so, could you please point me to it)?

Best regards, Roman

**Comment by snk4tr** [comment, marked as answer]
Hi @windj007! 
Thank you for your kind words about the paper and excuse us for the long response. 
You can see the computation strategy for the Distribution-based metrics in our [results_benchmark.py](https://github.com/photosynthesis-team/piq/blob/244865e3e80ec22ae075e7dfef8ea3ce10a55891/tests/results_benchmark.py#L288) script. Your understanding is correct: we split both images into crops and compute the distance between sets of crops the same way as we would do for sets of full images. Please let us know if anything is still unclear.

Best regards, Sergey

**Reply by windj007** [reply]
Thank you! It's clear now.
